scripts/create_strings.py

```diff
@@ -47,11 +50,79 @@ def generate_loading_pages(output_dir):
     )
 
 
-def create_resource_files(output_dir):
+def _find_string(lang, string):
+    from kolibri.main import initialize
+    from django.utils.translation import override
+    from django.utils.translation import ugettext as _
+    from django.utils.translation import to_locale
+
+    initialize(skip_update=True)
+
+    with override(lang):
+        new_string = _(string)
+        if new_string != string and new_string:
+            return new_string
+
+    for message_file in os.listdir(
+        resources.files("kolibri") / "locale" / "en" / "LC_MESSAGES"
+    ):
+        if message_file.endswith(".json"):
+            with open(
+                resources.files("kolibri")
+                / "locale"
+                / "en"
+                / "LC_MESSAGES"
+                / message_file,
+                "r",
+            ) as f:
+                messages = json.load(f)
+                for key, value in messages.items():
+                    if value == string:
+                        try:
+                            # Do this in case we have a legacy translation file - this should be cleaned up
+                            # in a future version of Kolibri
+                            with open(
+                                resources.files("kolibri")
+                                / "locale"
+                                / to_locale(lang)
+                                / "LC_MESSAGES"
+                                / message_file,
+                                "r",
+                            ) as f:
```

Comment:
might be better / safer to use a different var, since `f` is already defined?

Reply:
Seems wise!

Reply:
Updated!